lldb/source/Breakpoint/Watchpoint.cpp

```diff
@@ -211,6 +212,35 @@ bool Watchpoint::CaptureWatchedValue(const ExecutionContext &exe_ctx) {
   return (m_new_value_sp && m_new_value_sp->GetError().Success());
 }
 
+bool Watchpoint::WatchedValueReportable(const ExecutionContext &exe_ctx) {
+  if (!m_watch_modify)
+    return true;
+  if (!m_type.IsValid())
+    return true;
+
+  ConstString watch_name("$__lldb__watch_value");
```

Comment:
Seems like you can pass this directly to `ValueObjectMemory::Create` as a string literal and make @bulbazord happy?

Reply:
I've been reading through ValueObject and I'm not sure yet what the implications of removing ConstString from ValueObject are. If and when we can remove it from ValueObject, we'll be able to clean up this one too, so I would say don't worry about it right now.

As a suggestion, if you anticipate `WatchedValueReportable` being called many times, it would be less work to create that ConstString once as a `static` rather than creating one on each invocation.